historical_system_profiles/models.py

```diff
@@ -20,6 +20,7 @@ class HistoricalSystemProfile(db.Model):
         db.DateTime, default=datetime.utcnow, onupdate=datetime.utcnow
     )
     system_profile = db.Column(JSONB)
+    captured_on = _get_captured_date()
```

Comment:
The model objects have a specific format (see https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/). Basically in this area up top, you just define your columns (possibly w/ defaults) and then set the data later on in the `__init__`.

For this one, you could probably do something like
```python
captured_on =  db.Column(db.DateTime, default=datetime.utcnow)
```

That way we default to the created date/time if there's no value set.

We use alembic to autogenerate the actual SQL based on the model objects as well. After adding the field, you can run `prometheus_multiproc_dir=`mktemp -d` FLASK_APP=historical_system_profiles.app:get_flask_app_with_migration flask db migrate -m 'added captured_on column'` or similar and it should create a new file in the `migrations` directory that you can add to this commit. You can then run `flask db migrate` and it will apply the schema update to the database.